example/20-componentconfig-gardenlet.yaml

```
@@ -125,6 +125,7 @@ featureGates:
   ForceRestore: false
   DefaultSeccompProfile: true
   CoreDNSQueryRewriting: true
+  IPv6SingleStack: true
```

I don't think this should be enabled in this example file.

```suggestion
  IPv6SingleStack: false
```

---

It is enabled in the apiserver example per default aswell, thats why i made it here accordingly. Should i disable it in both cases?

---

For the API server, it is only used for validation purposes and only when the `ipFamily=IPv6` was set. Hence, it didn't cause "harm" so far. Perhaps it's indeed reasonable to disable it everywhere?